src/styles/variables.less

```diff
@@ -11,16 +11,22 @@
 @landing-offset-top:                                  (@landing-search-area-height + @navbar-height);
 @landing-search-area-height:                          52px;
 @landing-side-bar-bg:                                 #2C343D;
+@landing-side-bar-link-color:                         @color-pf-blue-300;
+@landing-side-bar-link-color-hover:                   lighten(@landing-side-bar-link-color, 10%);
 @landing-side-bar-top-offset:                         72px;
-@landing-side-bar-width-lg:                           425px;
-@landing-side-bar-width-md:                           375px;
+@landing-side-bar-width-lg:                           475px;
+@landing-side-bar-width-md:                           425px;
 @landing-side-bar-width-sm:                           325px;
 @navbar-height:                                       72px;
 @saas-hover:                                          fade(@color-pf-blue-700, 35%);
 @saas-border-bottom-color:                            fade(@color-pf-blue-700, 60%);
 @saas-text:                                           @color-pf-white;
 @search-area-border:                                  fade(@color-pf-blue-700, 60%);
-@screen-xl-min:                                       1800px;
+@screen-lg-max:                                       (@screen-xl-min - 1);
```

Comment:
Since [these same vars exist in console](https://github.com/openshift/origin-web-console/blob/master/app/styles/_variables.less#L104-L108), should these be moved to common so they're not declared twice?